src/main/webapp/WEB-INF/view/activityFeed.jsp

```diff
@@ -0,0 +1,21 @@
+<%@taglib prefix="t" tagdir="/WEB-INF/tags" %>
+<%--
+  Copyright 2017 Google Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+--%>
+    <t:base>
```

Comment:
`t` is the prefix of the library, and `base` is the name of the tag file we are using. Anything wrapped within these tags will be the JSP called by `<jsp:doBody/>` in `base.tag`